owl-bot-migration/migrate-to-owlbot.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
```

Comment:
When used this script for Logging, the `owlbot.py` generated had invalid syntax. It had incomplete `move` code from `synth.py`.

Reply:
Thanks, I'll take a look

Reply:
You're right. I had to manually delete two lines.

https://github.com/googleapis/google-cloud-php/pull/4838